src/helpers/figures/charts/chart_factory.ts

```diff
@@ -184,6 +187,18 @@ export function getSmartChartDefinition(zone: Zone, getters: Getters): ChartDefi
       legendPosition: newLegendPos,
     };
   }
+  const _dataSets = createDataSets(getters, dataSets, sheetId, dataSetsHaveTitle);
+  if (singleColumn && getData(getters, _dataSets[0]).every((e) => typeof e === "string")) {
+    return {
+      title,
```

Comment:
"A" is the title even though it's part of the dataset ?
![image](https://github.com/odoo/o-spreadsheet/assets/10908305/2e6c0a02-ffc8-4fa4-ab70-c18d7c2ebb9f)